core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -89,7 +89,7 @@ class GroupMetadataManager(brokerId: Int,
   private val openGroupsForProducer = mutable.HashMap[Long, mutable.Set[String]]()
 
   /* setup metrics*/
-  val partitionLoadSensor = metrics.sensor("PartitionLoadTime")
+  private val partitionLoadSensor = metrics.sensor("GroupPartitionLoadTime")
```

Comment:
nit: maybe we may as well add constant names for this as well as the metric group?